packages/text-annotator/test/index.html

```diff
@@ -287,12 +287,13 @@ <h3>Not annotatable block!</h3>
       window.onload = async () => {
         var contentContainer = document.getElementById('content');
 
-        const style = ((annotation, state, z) => ({
-          fillOpacity: 0.2,
-          underlineColor: '#00ff00',
-          underlineOffset: z * 3.5,
-          underlineThickness: 2
-        }));
+        const style = ((annotation, state, z) =>
+	        ({
+		        fillOpacity: 0.2,
+		        underlineColor: '#9b3c06',
+		        underlineOffset: (z * 3.5) || 0,
```

Comment:
Added handling for the case when `z` is not provided, which is relevant for the `CANVAS`

Reply:
Ah - I see you are using our new `z` parameter ;-) It's not really included in the TS function arg definition yet. But we'll keep it as a feature. Happy to see you use it, too :-)

Reply:
I didn't...
<img src="https://github.com/recogito/text-annotator-js/assets/68850090/3fe4e954-5d3c-46b3-87d6-68b6d4290aa6" width="333" />
(it was already in the test file 😅. But it was breaking the rendering of the underline when I changed the renderer from the `SPAN` to `CANVAS`. As latter doesn't pass the `z` arg to the styles function)